control-services/cli/config.py

```diff
@@ -148,12 +154,21 @@ def __init__(self, jsonConfigFile, envPath):
         cfg['jenkins']['casc_file_path'] = cfg['jenkins']['image_dir_path'] + 'casc.yaml'   \
 
         cfg['jenkins']['casc'] = {}
-        cfg['jenkins']['casc']['unclassified'] = {}        
+        cfg['jenkins']['casc']['unclassified'] = {}
         cfg['jenkins']['casc']['unclassified']['location'] = {}
         cfg['jenkins']['casc']['unclassified']['location']['url'] = {}
         cfg['jenkins']['casc']['unclassified']['location']['url'] =                                         \
             'http://' + cfg['host']['ip'] + '/' + cfg['jenkins']['port'] + '/'
 
+        cfg['jenkins']['casc']['authorizationStrategy'] = {}
```

Comment:
This is a block of code I had on the same machine, but it looks like it was swept up in this PR...

Reply:
Just removed that block and committed it